ShopApp/App/Modules/Account/Controllers/AccountViewController.swift

```diff
@@ -10,12 +10,13 @@ import UIKit
 
 import ShopApp_Gateway
 
-class AccountViewController: BaseViewController<AccountViewModel>, AccountTableProviderDelegate, AccountNotLoggedHeaderDelegate, AccountLoggedHeaderDelegate, AccountFooterDelegate {
+class AccountViewController: BaseViewController<AccountViewModel>, AccountTableProviderDelegate, AccountNotLoggedHeaderDelegate, AccountFooterDelegate {
     @IBOutlet private weak var tableView: UITableView!
     
     private var selectedPolicy: Policy?
     
     var tableProvider: AccountTableProvider!
+    var isOrdersEnabled: Bool!
```

Comment:
You inject this value to view controller and then pass it to the provider, but you can inject it directly to a provider